pkg/datapath/tables/node_address.go

```diff
@@ -57,6 +57,13 @@ type NodeAddress struct {
 	DeviceName string
 }
 
+func (n *NodeAddress) DeepEqual(other *NodeAddress) bool {
+	return n.Addr == other.Addr &&
```

Comment:
Add a comment to "NodeAddress" struct that if fields are added/modified this needs to be updated (or better yet, let's auto-generate these).